### PR TITLE
add audit log

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Review by whitewum — DISMISSED** [review]

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Audit logs
+
+The audit logs store all operations received by graph service in categories, then provide the logs for users to track specific types of operations as needed.
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:3]: The audit logs --？Nebula Graph audit logs?

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+|`util`|`SHOW HOSTS`、`SHOW USERS`、`SHOW ROLES`、`SHOW SNAPSHOTS`、`SHOW SPACES`、`SHOW PARTS`、`SHOW TAGS`、`SHOW EDGES`、`SHOW INDEXES`、`SHOW CREATE SPACE`、`SHOW CREATE TAG/EDGE`、`SHOW CREATE INDEX`、`SHOW INDEX STATUS`、`SHOW LISTENER`、`SHOW TEXT SEARCH CLIENTS`、`SHOW DRAINER CLIENTS`、`SHOW FULLTEXT INDEXES`、`SHOW CONFIGS`、`SHOW CHARSET`、`SHOW COLLATION`、`SHOW STATS`、`SHOW SESSIONS`、`SHOW META LEADER`、`SHOW DRAINERS`、`SHOW QUERIES`、`SHOW JOB`、`SHOW JOBS`、`DESCRIBE INDEX`、`DESCRIBE EDGE`、`DESCRIBE TAG`、`DESCRIBE SPACE`、`DESCRIBE USER`、`USE SPACE`、`SIGN IN TEXT SERVICE`、`SIGN OUT TEXT SERVICE`、`SIGN IN DRAINER SERVICE`、`SIGN OUT DRAINER SERVICE`、`EXPLAIN`、`PROFILE`、`KILL QUERY`、`DOWNLOAD HDFS`、`INGEST`|Logs the information about util statements. |
+|`unknown`|-|Logs the information about unrecognized statements.|
+
+## Configure audit logs
+
+You need to configure the configuration of the graph service to view audit logs. The default file path of configuration is `/usr/local/nebula/etc/nebula-graphd.conf`.
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:24]:
```suggestion
You need to configure the graph service file to view audit logs. The default file path of configuration is `/usr/local/nebula/etc/nebula-graphd.conf`.
```

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+!!! note
+
+    After modifying the configuration, you need to restart the graph service to take effect.
+
+Parameter descriptions are as follows:
+
+|Parameter|Predefined value|Description|
+|:--|:--|:--|
+| `enable_audit` | `false` | Whether or not to enable audit logs. |
+| `audit_log_handler` | `file` | Where the audit logs will be written. Optional values are `file`（local file） and `es`(Elasticsearch). |
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:35]:
```suggestion
| `audit_log_handler` | `file` | Where will the audit logs be written. Optional values are `file`（local file） and `es`(Elasticsearch). |
```

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+| `audit_log_file` | `./logs/audit/audit.log` | Takes effect only when `audit_log_handler=file`. The path for storing audit logs. The value can be absolute or relative. |
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:36]:
```suggestion
| `audit_log_file` | `./logs/audit/audit.log` | Takes effect only when `audit_log_handler=file` is set. The path for storing audit logs, the value of which can be absolute or relative. |
```

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+| `audit_log_strategy` | `synchronous` | Takes effect only when `audit_log_handler=file`. The strategy of the audit logs. Optional values are `asynchronous` and `synchronous`. When set to `asynchronous`, the log using memory buffer does not block the main thread, but the log may be missing due to insufficient cache. When set to `synchronous`, the log directly to file, flush and sync every event. |
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:37]:
```suggestion
| `audit_log_strategy` | `synchronous` | Sets the method to synchronize audit logs. Takes effect only when `audit_log_handler=file` is set.  Optional values are `asynchronous` and `synchronous`. When `asynchronous` is set, log events are cached in memory and do not block the main thread, but may result in missing logs due to insufficient cache. When `synchronous` is set, log events are refreshed and synchronized to the file each time. |
```

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+|`CATEGORY`| The category of the audit logs.|
+|`TIMESTAMP`| The generation time of the audit logs. |
+|`TERMINAL`| The reserved field.|
+|`CONNECTION_ID`| The session ID of the connection. |
+|`CONNECTION_STATUS`| The status of the connection. `0` indicates success, and other numbers indicate different error messages.|
+|`CONNECTION_MESSAGE`| An error message is displayed when the connection fails.|
+|`USER`| The user name of the connection. |
+|`CLIENT_HOST`| The IP address of the client.|
+|`HOST`| The IP address of the host. |
+|`SPACE`| The graph space to perform the query.|
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:97]:
```suggestion
|`SPACE`| The graph space where you perform queries.|
```

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+|Field|Description|
+|:--|:--|
+|`CATEGORY`| The category of the audit logs.|
+|`TIMESTAMP`| The generation time of the audit logs. |
+|`TERMINAL`| The reserved field.|
+|`CONNECTION_ID`| The session ID of the connection. |
+|`CONNECTION_STATUS`| The status of the connection. `0` indicates success, and other numbers indicate different error messages.|
+|`CONNECTION_MESSAGE`| An error message is displayed when the connection fails.|
+|`USER`| The user name of the connection. |
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:94]: The user name for the Nebula Graph connection?

File: docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+Parameter descriptions are as follows:
+
+|Parameter|Predefined value|Description|
+|:--|:--|:--|
+| `enable_audit` | `false` | Whether or not to enable audit logs. |
+| `audit_log_handler` | `file` | Where the audit logs will be written. Optional values are `file`（local file） and `es`(Elasticsearch). |
+| `audit_log_file` | `./logs/audit/audit.log` | Takes effect only when `audit_log_handler=file`. The path for storing audit logs. The value can be absolute or relative. |
+| `audit_log_strategy` | `synchronous` | Takes effect only when `audit_log_handler=file`. The strategy of the audit logs. Optional values are `asynchronous` and `synchronous`. When set to `asynchronous`, the log using memory buffer does not block the main thread, but the log may be missing due to insufficient cache. When set to `synchronous`, the log directly to file, flush and sync every event. |
+| `audit_log_max_buffer_size` | `1048576` |Take effect only when `audit_log_handler=file` and `audit_log_strategy=asynchronous`. The size of the memory buffer used for logging. Unit: bytes.  |
```
Comment (abby-cyber) [inline docs-2.0/5.configurations-and-logs/2.log-management/audit-log.md:38]: ditto 
pls check below content

**Review by abby-cyber — DISMISSED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**